test/common/mongoose_helper.erl

```diff
@@ -499,6 +500,22 @@ restart_listener(Spec, Listener) ->
     rpc(Spec, mongoose_listener, stop_listener, [Listener]),
     rpc(Spec, mongoose_listener, start_listener, [Listener]).
 
+change_listener_idle_timeout(ListenerPort, Timeout) ->
+    ListenerConfig = get_listener_config_by_port(ListenerPort),
+    #{protocol := ProtocolOpts} = ListenerConfig,
+    % Chage the default idle_timeout to 5s to test if sse will override it
```

Comment:
Typo: 'chage'. It is also not 5s, please update or just remove this comment, because for me it does not add much value.

Reply:
Removed